src/toil/test/provisioners/aws/awsProvisionerTest.py

```diff
@@ -220,7 +220,7 @@ def _test(self, preemptibleJobs=False):
 
 
 @integrative
-@needs_mesos
+# @needs_mesos
```

Comment:
Doesn't this still need Mesos?
```suggestion
@needs_mesos
```